app/views/assignments/_grades.html.erb

```diff
@@ -8,14 +8,26 @@
     </thead>
     <tbody>
       <% max_points = 0 %>
-      <% @assignment.rubric_fields.each do |field| %>
-      <% grading_field = @submission.grading_fields.where(rubric_field_id: field.id).first %>
-      <% max_points += field.points.to_i %>
-        <tr>
-          <td><%= field.name %></td>
-          <td><%= grading_field.score %></td>
-          <!-- <td><%= field.points %></td> -->
-        </tr>
+      <% if @assignment.id == 1 %>
```

Comment:
I accidentally already ordered assignment 1 by update date (so sorting by id is out of order for assignment 1)